services/worker/tests/fixtures/hub.py

```diff
@@ -568,4 +597,12 @@ def hub_datasets(
             "first_rows_response": create_first_rows_response(hub_public_big, BIG_cols, BIG_rows),
             "parquet_and_dataset_info_response": None,
         },
+        "external_files": {
+            "name": hub_public_external_files,
+            "config_names_response": create_config_names_response(hub_public_external_files),
+            "split_names_response": create_split_names_response(hub_public_external_files),
+            "splits_response": create_splits_response(hub_public_external_files),
+            "first_rows_response": create_first_rows_response(hub_public_external_files, TEXT_cols, TEXT_rows),
+            "parquet_and_dataset_info_response": None,
```

Comment:
not sure if this was needed - I added it anyway

Reply:
no, not needed since we never access this dataset from the `hub_datasets` fixture. But we could, to ensure we know how to manage datasets with a script (all the other tests use a simple data file).

Reply:
I'll leave it here in case it can be useful for some tests later if you don't mind

Reply:
Can you create an issue ([here](https://www.notion.so/huggingface2/Internal-issues-b15b180c923346ceb6893d1e6805f180) or [here](https://github.com/huggingface/datasets-server/)) to ensure we remind it?

Reply:
done on notion :)